multimodal/src/autogluon/multimodal/models/mmdet_image.py

```diff
@@ -341,8 +344,50 @@ def get_layer_ids(
         Basically, id gradually increases when going from the output end to
         the input end. The layers defined in this class, e.g., head, have id 0.
 
-        Setting all layers as the same id 0 for now.
-        TODO: Need to investigate mmdetection's model definitions
+        Returns
+        -------
+        A dictionary mapping the layer names (keys) to their ids (values).
+        """
+
+        # two stage lr: backbone v.s. else, or head v.s. else
+        if self.backbone_layers:
+            return self.get_layer_ids_by_backbone()
+        else:
+            return self.get_layer_ids_by_head()
```

Comment:
Why needs to change the logic of getting layer ids? The `two_stage` lr choice doesn't rely on layer ids.

Reply:
In mmdet_model, we assign `self.name_to_id = self.get_layer_ids()` and `self.head_layer_names = [n for n, layer_id in self.name_to_id.items() if layer_id <= 0]`